Source/Core/DolphinQt/TAS/TASInputWindow.cpp

```diff
@@ -81,46 +91,40 @@ int TASInputWindow::GetTurboReleaseFrames() const
   return m_turbo_release_frames->value();
 }
 
-TASCheckBox* TASInputWindow::CreateButton(const QString& text, std::string_view group_name,
-                                          std::string_view control_name, InputOverrider* overrider)
+TASCheckBox* TASInputWindow::CreateButton(const QString& name)
```

Comment:
Do we have a good understanding of the implications of this change? What's an InputOverrider and are we sure we can just do away with it?

Reply:
InputOverrider as far as I know is only used in GBATASInputWindow